sdk/eventhub/Microsoft.Azure.EventHubs/tests/Infrastructure/TestUtility.cs

```diff
@@ -130,18 +150,50 @@ private static string ReadEnvironmentVariable(string variableName)
             return environmentVar;
         }
 
-        private static EventHubScope.AzureResourceProperties CreateNamespace() =>
-            Task
+        private static string ReadOptionalEnvironmentVariable(string variableName) => Environment.GetEnvironmentVariable(variableName);
+
+        private static EventHubScope.AzureResourceProperties CreateNamespace()
+        {
+            var environmentConnectionString = ReadOptionalEnvironmentVariable(TestConstants.EventHubsNamespaceConnectionStringEnvironmentVariable);
+
+            if (!string.IsNullOrEmpty(environmentConnectionString))
```

Comment:
According to Jared Parsons, this is the new hotness. `environmentConnectionString is { Length: > 0 }`
:D
https://twitter.com/jaredpar/status/1288547233718087680 :D

Reply:
You no can haz C# 9 yet!